.gitignore

```diff
@@ -21,3 +21,6 @@
 
 # virtual machine crash logs, see http://www.java.com/en/download/help/error_hotspot.xml
 hs_err_pid*
+
+#IntellijIdea
```

Comment:
/.idea/
/target/
*.log
*.iml

Reply:
*.log is on the 5 line and *.iml on the 26